TodoListManagerTest/src/test/resources/ProjectsStory/add_project.feature

```diff
@@ -0,0 +1,39 @@
+@tag
+  Feature: add new project
+    As a user, I want to add a new project to manage my work.
+
+  @tag1
+  Scenario Outline: Add a project with all fields
+    Given the URL https://localhost:4567/projects
+    When user creates a project with "<title>", "<completed>", "<active>" and "<description>"
+    And passes that project in a post request
+    Then success error code is returned
+    And object body is returned
+
+    Examples:
+      | title |completed | active | description |
+      | Rainbow | false | true | Rain |
+      | planting | false | false | soil|
+
+  @tag2
+  Scenario Outline: Add a project without fields
+    Given the URL https://localhost:4567/projects
+    When user creates a project
+    And passes that project in a post request
+    Then success error code is returned
+    And object body is returned
+
+    Examples:
+    ||
+    ||
```

Comment:
you need a proper example

Reply:
Since it is adding a project without fields, there should have no parameters. I have add a comment in the program to illustrate this.